### PR TITLE
Updating references to ContinuumIO -> intake

### PR DESCRIPTION
**Pull request by jsignell**
I think I got them all with the exception of gitter which doesn't seem to have moved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Intake: A general interface for loading data
 
-![Logo](https://github.com/ContinuumIO/intake/raw/master/logo-small.png)
+![Logo](https://github.com/intake/intake/raw/master/logo-small.png)
 
-[![Build Status](https://travis-ci.org/ContinuumIO/intake.svg?branch=master)](https://travis-ci.org/ContinuumIO/intake)
-[![Coverage Status](https://coveralls.io/repos/github/ContinuumIO/intake/badge.svg?branch=master)](https://coveralls.io/github/ContinuumIO/intake?branch=master)
+[![Build Status](https://travis-ci.org/intake/intake.svg?branch=master)](https://travis-ci.org/intake/intake)
```
Comment (martindurant) [inline README.md]: This one breaks - Travis is still watching ContinuumIO, which is probably fine to leave as it is.

Comment (jsignell) [reply README.md]: I don't see it as broken

Comment (martindurant) [reply README.md]: Sorry, must have been some kind of lag

**Comment by martindurant** [comment]
The failure is on the DAT cache, the test dataset seems to have disappeared

**Comment by martindurant** [comment]
Can you just set intake/source/tests/test_cache::test_dat to xfail for now?

**Comment by martindurant** [comment]
Seems like the way to get DAT to successfully find files, download and exit is to host them on the same machine... will look into changing the test to do that later.

**Comment by jsignell** [comment]
Ok I'll let it fail

**Pull request merged**